_sass/blog.scss

```diff
@@ -45,7 +45,6 @@
 		overflow: hidden;
 		a {
 			display: block;
-			width: 50%;
```

Comment:
Why was this removed?

Reply:
@asubedy navigation has to span the entire width of the container
With width:50%
![image](https://user-images.githubusercontent.com/55309324/180595587-1afe9e95-6123-43fe-aeb7-d1087dcb9a28.png)
When removed
![image](https://user-images.githubusercontent.com/55309324/180595599-102ff836-423c-4102-bccc-eb7144820e8c.png)

Reply:
Oh ok nice nice. 👍